### ISSUE TITLE
Keyboard navigation does not work on loading indicators

### ISSUE THREAD

**Issue by rugk** [mastodon/mastodon#15302]
### Expected behaviour

<kbd>↓</kbd> should focus the ` …`  button.

### Actual behaviour

<kbd>↓</kbd> does nothing.

### Steps to reproduce the problem

1. Use your keyboard <kbd>↓</kbd> (down arrow key) to move one column with toots.
2. Go down until you reach this "gap" on loading indicator:
![grafik](https://user-images.githubusercontent.com/11966684/101403660-d8b71a00-38d5-11eb-8214-1ddedae30a07.png)

### Specifications

v3.2.1

Firefox 83
Linux/Fedora 33/GNOME

May also be related to https://github.com/tootsuite/mastodon/issues/13200

**Comment by diondiondion** [comment]
Fixed via #35455
